### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jukent**
First pass at updating the README for the tutorial content repository.

Some to do steps hinted at:
- Create a fleshed out contributor's guide
- Create a notebook and md template
- Set up continuous integration

Do we want the document tree to be updated here with tutorials or to have a separate file (perhaps a README in the tutorials directory) that lists all tutorials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,49 @@
...
 
 A single repository to host community-contributed Pythia tutorial content (notebooks, data, and markdown).
+
+The goal of this repository is to create a common location and environment for Pythia Python tutorials. For the Python learner, this has the benefits of consistent installation / spin up before jumping in to each lesson. For the Pythia team, this ownership allows us to maintain content long-term so that we can fix things when they break.
+
+We, the Pythia team, want there to be a low barrier to entry for the tutorial speaker to develop and host their tutorial content in this repository. For the speaker's convenience we will provide a simple template. This repository differs from `pythia-foundations` book, which has a high barrier for entry, with its strict and consistent style guide and rigorous review process before merging. Further, the tutorial notebooks in this repository are designed to be accompanied by a live speaker or recording. This means that the amount of explanatory text between code cells or excercises may differ between tutorials based on the tutorial speaker and developer's preferred style.
```
Comment (mgrover1) [inline README.md:9]: perhaps change "want there to be a low barrier to entry for the tutorial speaker" to "we want to lower the barrier of entry for the tutorial speaker"?

File: README.md [hunk 2]
```diff
@@ -3,3 +3,49 @@
...
+   |    |- ANOTHER_TOPIC_YYYY_MM_DD/
+   |    |    |-  `topic.ipynb`
+   |    |    |-  `README.md`
+   |-  `environment.yaml`
+   |-  `README.md`
+```
+
+Some things to notice:
+- There is a single `data` directory across all tutorials. This is designed to encourage re-use of datasets.
+- Within the `tutorials` direcotry there is a unique directory for every tutorial and a `template` directory, containing a markdown and a notebook template (coming soon).
```
Comment (mgrover1) [inline README.md:37]: Watch the spelling of "directory"

File: README.md [hunk 3]
```diff
@@ -3,3 +3,49 @@
...
+   |    |    |-  `topic.ipynb`
+   |    |    |-  `README.md`
+   |-  `environment.yaml`
+   |-  `README.md`
+```
+
+Some things to notice:
+- There is a single `data` directory across all tutorials. This is designed to encourage re-use of datasets.
+- Within the `tutorials` direcotry there is a unique directory for every tutorial and a `template` directory, containing a markdown and a notebook template (coming soon). 
+- Each tutorial is named with its topic and date with the format `TOPIC_YYYY_MM_DD`.
```
Comment (mgrover1) [inline README.md:38]: Provide an example here (ex. `pandas_2021_08_03`)

**Review by mgrover1 — CHANGES_REQUESTED** [review]
Looks good - just a few minor changes

**Review by clyne — APPROVED** [review]
Looks great, Julia! Thanks for doing this.

**Review by mgrover1 — APPROVED** [review]
Looks great! Thanks @jukent

**Pull request merged**